CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 1.0.2 - 2021-05-05
```

Comment:
Sorry couldn’t resist using the only internationally approved standard for dates here :)

Reply:
Also, not sorry